index.js

```diff
@@ -99,12 +99,32 @@ module.exports = {
 
     // We shouldn't enforce filename casing to always be the same.  e.g. We may
     // prefer kebab-case for bin files, PascalCase for React components, etc.
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/filename-case.md
     'unicorn/filename-case': 'off',
 
     // Uses safe-regex to disallow potentially catastrophic exponential-time
     // regular expressions.
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-unsafe-regex.md
     'unicorn/no-unsafe-regex': 'error',
 
+    // This rule is onerous in React functional components, has been debated
+    // by the community and proven as an unnecessary optimization
```

Comment:
Is there a link to support this claim? I'd wager it'd be an issue in https://github.com/sindresorhus/eslint-plugin-unicorn ?

Reply:
A single link? No. Feel free to google. There's no shortage of discussion on the topic.

Reply:
The React team specifically have spoken out against this optimization. People familiar with Javascript workings have spoken out against this optimizations.

The people who claimed that it reduced performance were found to be measuring using the development version of React, not production. This is highly unlikely to be the root cause of performance issues under most circumstances.

This is the Javascript equivalent of the anti-vaxxer movement. The source of it has been discredited but yet the myth persists.